### PR TITLE
Add README documentation for scanner remote options

### PR DESCRIPTION
**Pull request by sambetts-cisco**

### CODE REVIEW

**Review by pbalogh-sa — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -298,6 +298,72 @@ SCANNERS_LIST="grype" BACKEND_HOST=localhost:9999 BACKEND_DISABLE_TLS=true kubec
 ANALYZER_LIST="syft" kubeclarity-cli analyze nginx:latest -o nginx.sbom --merge-sbom inputsbom.xml
 ```
 
+### Remote scanner servers
+
+When running the kubeclarity CLI to scan for vulnerabilties the CLI will need
```
Comment (akpsgit) [inline README.md:303]: to scan for vulnerabilities, the

File: README.md [hunk 2]
```diff
@@ -298,6 +298,72 @@ SCANNERS_LIST="grype" BACKEND_HOST=localhost:9999 BACKEND_DISABLE_TLS=true kubec
...
+to download the relevant vulnerablity DBs to the location where the kubeclarity
+CLI is running. If running the CLI in a CI/CD pipeline this might mean having
```
Comment (akpsgit) [inline README.md:305]: Running the CLI in a CI/CD pipeline will result in downloading the DBs in each run, wasting time and bandwidth.

File: README.md [hunk 3]
```diff
@@ -298,6 +298,72 @@ SCANNERS_LIST="grype" BACKEND_HOST=localhost:9999 BACKEND_DISABLE_TLS=true kubec
 ANALYZER_LIST="syft" kubeclarity-cli analyze nginx:latest -o nginx.sbom --merge-sbom inputsbom.xml
 ```
 
+### Remote scanner servers
```
Comment (akpsgit) [inline README.md:301]: Maybe we should put this information in a separate readme and point to it from the main one? I think that this is relevant for a small percent of the users. Maybe better to keep the readme only with the essential instructions (installation and cli instructions)?

File: README.md [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
+
+- [Why?](#why)
+  - [SBOM & Vulnerability detection challenges](#sbom--vulnerability-detection-challenges)
+  - [Solution](#solution)
+- [Features](#features)
+  - [Integrated SBOM generators and vulnerability scanners](#integrated-sbom-generators-and-vulnerability-scanners)
+- [Architecture](#architecture)
+- [Getting Started](#getting-started)
+  - [KubeClarity Backend](#kubeclarity-backend)
+    - [Install using Helm](#install-kubeclarity-in-a-k8s-cluster-using-helm)
```
Comment (FrimIdan) [inline README.md:17]: it should be `#install-using-helm`

File: README.md [hunk 5]
```diff
@@ -1,10 +1,39 @@
...
+    - [Exporting results to KubeClarity Backend](#exporting-results-to-kubeclarity-backend)
+- [Advanced Configuration](#advanced-configuration)
+  - [SBOM generation using local docker image as input](#sbom-generation-using-local-docker-image-as-input)
+  - [Vulnerability scanning using local docker image as input](#vulnerability-scanning-using-local-docker-image-as-input)
+  - [Private Registry Support For CLI](#private-registry-support-for-cli)
+  - [Private Registry Support For K8s Runtime Scan](#private-registries-support-for-k8s-runtime-scan)
+  - [Merging of SBOM and vulnerabilities across different CI/CD stages](#merging-of-sbom-and-vulnerabilities-across-different-cicd-stages)
+  - [Remote Scanner Servers For CLI](#remote-scanner-servers-for-cli)
+- [Limitations](#limitations)
+- [Contributing](#contributing)
```
Comment (FrimIdan) [inline README.md:32]: you missed `[Roadmap](#roadmap)`

File: README.md [hunk 6]
```diff
@@ -343,19 +398,93 @@ Note:
...
+# For example:
+ANALYZER_LIST="syft" kubeclarity-cli analyze nginx:latest -o nginx.sbom --merge-sbom inputsbom.xml
+```
+
+## Remote Scanner Servers For CLI
+
+When running the kubeclarity CLI to scan for vulnerabilties, the CLI will need
+to download the relevant vulnerablity DBs to the location where the kubeclarity
+CLI is running. Running the CLI in a CI/CD pipeline will result in downloading
+the the DBs on each run, wasting time and bandwidth. For this reason several of
```
Comment (akpsgit) [inline README.md:414]: the the

**Review by akpsgit — DISMISSED** [review]

**Review by pbalogh-sa — APPROVED** [review]

**Pull request merged**